pkg/kubectl/cmd/autoscale.go

```diff
@@ -54,7 +54,7 @@ func NewCmdAutoscale(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 		},
 	}
 	cmdutil.AddPrinterFlags(cmd)
-	cmd.Flags().String("generator", "horizontalpodautoscaler/v1beta1", "The name of the API generator to use. Currently there is only 1 generator.")
```

Comment:
Not sure if we can delete the generator.
cc @bgrant0607 to confirm.

We can let the old generator remain, which can continue to generate extensions.v1beta1.HPA and add a new generator to generate autoscaling.v1.HPA.
We can delete the v1beta1 generator when we delete extensions.v1beta.HPA.